azerothcore/azerothcore-wotlk
# Fixed Nagrand Arena door position

Fixed Nagrand Arena door position, thx @Shard-MW 

##### CHANGES PROPOSED:
- change the coords  of the Nagrand Arena doors

##### ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/2517

##### TESTS PERFORMED:
Nagrand Arena doors **BEFORE this PR**:

![nagrand arena door](https://user-images.githubusercontent.com/30301841/71256002-d14ad980-232f-11ea-87e8-e3e4fd176f67.PNG)
![nagrand arena door](https://user-images.githubusercontent.com/519778/71321870-303f5880-24c0-11ea-9164-9c689688e1a6.png)  
  
Nagrand Arena doors (both) **after this PR:**
![immagine](https://user-images.githubusercontent.com/519778/71323004-afd42400-24ce-11ea-9454-ab65ad3f327c.png)
![immagine](https://user-images.githubusercontent.com/519778/71323018-d72af100-24ce-11ea-816f-7d4e384d0720.png)
  

##### HOW TO TEST THE CHANGES:
```
.debug arena
- join arena 2v2 skirmish with 2 players
- test Nagrand Arena and check if thhe gameobject still appear on the floor
```

If you want to be sure that joinin arena 2v2 you go into the Nagrand Arena add in `BattlegroundMgr.cpp` at line 438 `bgTypeId = BATTLEGROUND_NA`

##### Target branch(es):

## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
